core/src/test/java/sem/group15/bubblebobble/core/objects/FilledBubbleTest.java

```diff
@@ -73,7 +79,7 @@ public void testHandleCollisionPlayerSlow(){
      */
     @Test
     public void testDraw() {
-        SpriteBatch batch = Mockito.mock(SpriteBatch.class);
+        SpriteBatch batch = mock(SpriteBatch.class);
         Texture texture = null;
```

Comment:
why did you change Mockito.mock to mock everywhere, just for layout cleanness? <layout>